src/services/WalletStateSchema.test.ts

```diff
@@ -611,11 +611,11 @@ describe("The current WalletStateSchema version", () => {
-		container2 = await addNewCredentialEvent(container2, "cred2y", "", "");
+		container2 = jsonParseTaggedBinary("{\"lastEventHash\":\"\",\"events\":[{\"schemaVersion\":1,\"eventId\":1950214055,\"parentHash\":\"\",\"timestampSeconds\":0,\"type\":\"new_credential\",\"credentialId\":4048680674,\"data\":\"cred1\",\"format\":\"\",\"kid\":\"\",\"batchId\":0,\"credentialIssuerIdentifier\":\"\",\"credentialConfigurationId\":\"\",\"instanceId\":0},{\"schemaVersion\":1,\"eventId\":1458724476,\"parentHash\":\"4cb7a44642b3d07a8b521878fa892a38047db4dc2cdc7997a6366bffe9b234f7\",\"timestampSeconds\":1,\"type\":\"new_credential\",\"credentialId\":1776982089,\"data\":\"cred2\",\"format\":\"\",\"kid\":\"\",\"batchId\":0,\"credentialIssuerIdentifier\":\"\",\"credentialConfigurationId\":\"\",\"instanceId\":0},{\"schemaVersion\":1,\"eventId\":3856394471,\"parentHash\":\"8e560270b533d63fc32019eb3b4b4f43c0ad3332cca4f1967e61f82f00fb1fe2\",\"timestampSeconds\":2,\"type\":\"delete_credential\",\"credentialId\":4048680674},{\"schemaVersion\":1,\"eventId\":2509657177,\"parentHash\":\"57ffbdbd82ffd7fb0edd8b3d87260dc315c80ecece7f8a2864a9285769acfe3a\",\"timestampSeconds\":20,\"type\":\"delete_credential\",\"credentialId\":1776982089},{\"schemaVersion\":1,\"eventId\":3947301554,\"parentHash\":\"416d0b4c67622465202f40891bc9ab618671ca804af3f01bf9d9c18087436620\",\"timestampSeconds\":21,\"type\":\"new_credential\",\"credentialId\":3745183927,\"data\":\"cred2x\",\"format\":\"\",\"kid\":\"\",\"batchId\":0,\"credentialIssuerIdentifier\":\"\",\"credentialConfigurationId\":\"\",\"instanceId\":0},{\"schemaVersion\":2,\"eventId\":1390895745,\"parentHash\":\"cb219d53277b49d2f883b96d4fc294040df2c0e5156efb82fd7c5472cc6fb557\",\"timestampSeconds\":1761814188,\"type\":\"new_credential\",\"credentialId\":3374898187,\"data\":\"cred2y\",\"format\":\"\",\"kid\":\"\",\"batchId\":0,\"credentialIssuerIdentifier\":\"\",\"credentialConfigurationId\":\"\",\"instanceId\":0}],\"S\":{\"schemaVersion\":1,\"credentials\":[],\"presentations\":[],\"keypairs\":[],\"credentialIssuanceSessions\":[],\"settings\":{\"openidRefreshTokenMaxAgeInSeconds\":\"0\"}}}");
```

Comment:
These fossilized states can skip all the steps before the last `jsonParseTaggedBinary` since any preceding steps would be overwritten anyway. Meta-PR #913 would fix this.

Reply:
Fixed by #913.